db/data/20190313120752_delete_council_tax_from_transaction_types.rb

```diff
@@ -0,0 +1,9 @@
+class DeleteCouncilTaxFromTransactionTypes < ActiveRecord::Migration[5.2]
+  def up
+    TransactionType.find_by(name: 'council_tax')&.destroy
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration
```

Comment:
Why is this irreversible? If the up destorys, then all the down has to do is create:
```
def down
  TransactionType.create!(name: 'council_tax', operation: 'debit')
  TransactionType.populate_records # to update sorting.
end
```